exchanges/order/limits.go

```diff
@@ -0,0 +1,376 @@
+package order
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/shopspring/decimal"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+)
+
+var (
+	// ErrExchangeLimitNotLoaded defines if an exchange does not have minmax
+	// values
+	ErrExchangeLimitNotLoaded = errors.New("exchange limits not loaded")
+	// ErrPriceExceedsMin is when the price is lower than the minimum price
+	// limit accepted by the exchange
+	ErrPriceExceedsMin = errors.New("price exceeds minimum limit")
```

Comment:
The errors that have minimum requirements should be worded slightly different. In the case of this error the price is below the minimum limit yet we claim it exceeds it

Reply:
Pedantic nit but +1 on this:

```
submitorder --exchange=binance --pair=ETH-BTC --asset=spot --side=SELL --type=LIMIT --amount=0.0000001 --price=5000
2021/03/23 10:47:46 rpc error: code = Unknown desc = order manager: exchange binance unable to place order: amount exceeds minimum limit min: 0.001000 supplied 0.000000 for spot ETHBTC
```
"amount below minimum limit" rather than "amount exceeds minimum limit"